### PR TITLE
Remove reporter user

### PR DESCRIPTION
**Pull request by jsf9k**
Although not strictly necessary, this change will tighten security after dhs-ncats/cyhy_amis#153 is merged.  This pull request is dependent on dhs-ncats/cyhy_amis#153, so do not merge this pull request until that one is merged.

Note also that this will impact the code that @KyleEvers maintains that copies this data to the MOE.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
✌️

**Comment by jsf9k** [comment]
@KyleEvers I want to merge this in, but note that it will break the way you are currently accessing the Elasticsearch DB.  Can you instead access it using an instance role, as I am now doing?  See [here](https://github.com/cisagov/cyhy_amis/blob/develop/terraform/bod_docker_ec2.tf#L69-L91) and [here](https://github.com/cisagov/cyhy_amis/blob/develop/terraform/bod_docker_ec2.tf#L142).

I won't merge this until you change the code on your end.

**Comment by jsf9k** [comment]
This can be merged once cisagov/cyhy-feeds#19 and cisagov/cyhy_amis#184 are merged and deployed.

**Pull request merged**
